agenda-webapps/src/main/webapp/vue-app/agenda/components/AgendaToolbar.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <v-toolbar flat color="white">
+      <v-btn
+        outlined
+        class="mr-4"
+        color="grey darken-2"
+        @click="setToday">
+        Today
+      </v-btn>
+      <v-btn
+        fab
+        text
+        small
+        color="grey darken-2"
+        @click="prevDate">
+        <v-icon small>mdi-chevron-left</v-icon>
+      </v-btn>
+      <v-btn
+        fab
+        text
+        small
+        color="grey darken-2"
+        @click="nextDate">
+        <v-icon small>mdi-chevron-right</v-icon>
+      </v-btn>
+      <v-toolbar-title v-if="$refs.calendar">
+        {{ $refs.calendar.title }}
```

Comment:
You don't have a `$refs.calendar` anymore here, please get tile from parent component `Agenda.vue` that will retrieve generic data such as title of the agenda app.